week1/index.html

```diff
@@ -0,0 +1,129 @@
+          <img src="https://www.mcdonalds.co.kr/upload/product/pcfile/1583727649835.png" alt="맥치킨 모짜렐라 사진">
+          <div class="burger_content">
+            <h2 class="buger_name">맥치킨모짜렐라</h2>
+            <div class="burger_price">5,500</div>
+            <p class="buger_desc">송픽 3위 매콤짭짤 교과서</p>
+          </div>
+        </article>
+        <article class="burger_card">
+          <img src="https://www.mcdonalds.co.kr/upload/product/pcfile/1617184718172.png" alt="더블 필레오 피쉬 사진">
+          <div class="burger_content">
+            <h2 class="buger_name">더블필레오피쉬</h2>
+            <div class="burger_price">5,500</div>
+            <p class="buger_desc">뭔가 어취날 것 같음</p>
+          </div>
+        </article>
+        <article class="burger_card">
+          <img src="https://www.mcdonalds.co.kr/upload/product/pcfile/1621834846208.png" alt="트리플 치즈 버거 사진">
+          <div class="burger_content">
+            <h2 class="buger_name">트리플치즈</h2>
+            <div class="burger_price">5,500</div>
+            <p class="buger_desc">콜라 라지로 시켜야할 것 같음</p>
+          </div>
+        </article>
+        <article class="burger_card">
+          <img src="https://www.mcdonalds.co.kr/upload/product/pcfile/1644571675478.png" alt="더블빅맥 사진">
+          <div class="burger_content">
+            <h2 class="buger_name">더블빅맥</h2>
+            <div class="burger_price">5,500</div>
+            <p class="buger_desc">입찢어질 것 같은 버거</p>
+          </div>
+        </article>
+        <article class="burger_card">
+          <img src="https://www.mcdonalds.co.kr/upload/product/pcfile/1599119647587.png" alt="1955 사진">
+          <div class="burger_content">
+            <h2 class="buger_name">1955</h2>
+            <div class="burger_price">5,500</div>
+            <p class="buger_desc">저 구운양파 좀 이상하게 생김</p>
+          </div>
+        </article>
+        <article class="burger_card">
+          <img src="https://www.mcdonalds.co.kr/upload/product/pcfile/1648631643102.png" alt="아라비아따 사진">
+          <div class="burger_content">
+            <h2 class="buger_name">아라비아따</h2>
+            <div class="burger_price">5,500</div>
+            <p class="buger_desc">무슨 신메뉴</p>
+          </div>
+        </article>
+        <article class="burger_card">
+          <img src="https://www.mcdonalds.co.kr/upload/product/pcfile/1583727313256.png" alt="더블불고기 사진">
+          <div class="burger_content">
+            <h2 class="buger_name">더블불고기</h2>
+            <div class="burger_price">5,500</div>
+            <p class="buger_desc">냠냠굿</p>
+          </div>
+        </article>
+      </section>
+      <aside class="cart">
+        <section class="cart_card">
+          <h2>장바구니</h2>
+          <hr class="white">
+          <h2>누적금액</h2>
+          <h3>0원</h3>
+          <hr class="white">
+          <div class="cart_button">
+            <button class="order">주문하기</button>
+            <button class="cancel">취소하기</button>
+          </div>
+        </section>
+      </aside>
```

Comment:
```suggestion
        <section class="cart_card">
          <h2>장바구니</h2>
          <hr class="white">
          <h2>누적금액</h2>
          <p>0원</p>
          <hr class="white">
          <div class="cart_button">
            <button class="order">주문하기</button>
            <button class="cancel">취소하기</button>
          </div>
        </section>
```
1. 이 부분은 aside테그보단 section테그가 더 잘 어울리는 것 같습니다! 사이드메뉴같은 느낌보다 하나의 구획인 느낌이라, section테그로 박스를 만들고 main테그에서 이 박스의 위치를 정의해주는 게 좋을 것 같아요!
2. 금액 정보는 p테그가 조금 더 적절할 것 같습니다!